python/python/glide/async_commands/server_modules/ft_options/ft_create_options.py

```diff
@@ -276,14 +258,8 @@ def __init__(
         super().__init__(dimensions, distance_metric, type)
         self.initial_cap = initial_cap
 
-    def toArgs(self) -> List[TEncodable]:
-        """
-        Get the arguments representing the vector field created with FLAT algorithm.
-
-        Returns:
-            List[TEncodable]: A list of FLAT algorithm type vector arguments.
-        """
-        args = super().toArgs()
+    def to_args(self) -> List[TEncodable]:
+        args = super().to_args()
```

Comment:
Why removing docs?
You can change to dev docs and leave something simple

Reply:
This is an inherited class, and the doc from the main abstract class for this method is available when we hover over the method in IDE.

Reply:
If the docs in the abstract are sufficient, no need